helpdesk_mgmt_maintenance/models/maintenance_equipment.py

```diff
@@ -0,0 +1,35 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    ticket_ids = fields.Many2many('helpdesk.ticket', string = "Tickets")
```

Comment:
python attributes should be defined without spaces:
```suggestion
    ticket_ids = fields.Many2many('helpdesk.ticket', string="Tickets")
```